build_mode.hh

```diff
@@ -0,0 +1,27 @@
+
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#ifndef stringify
+#define _stringify(x) #x
+#define stringify(x) _stringify(x)
+#endif
```

Comment:
It's not good practice to use lowercase names for macros, it can lead to strange results if they are used in an unexpected context.

Reply:
Fixed in da321a894